src/app/shared/wysiwyg/prosemirror.markdown.editor.ts

```diff
@@ -88,49 +88,37 @@ export class ProseMirrorMarkdownEditor {
     return config;
   }
 
-  plainTextPlusLinks() {
-    // remove unsupported marks
-    let transaction = this.view.editor.state.tr
-      .removeMark(0, this.view.editor.state.doc.nodeSize - 2, markdownSchema.marks.strong)
-      .removeMark(0, this.view.editor.state.doc.nodeSize - 2, markdownSchema.marks.em)
-      .removeMark(0, this.view.editor.state.doc.nodeSize - 2, markdownSchema.marks.code);
-
-    this.view.updateState(this.view.editor.state.apply(transaction));
-
-    /* This removes images but also removes links too :( don't use this
-    this.view.updateState(this.view.editor.state.apply(this.view.editor.state.tr.clearMarkup(0, this.view.editor.state.doc.nodeSize - 2)));
-    */
-
-    let content = [];
+  plainTextWithLinks() {
+    let content = '';
     const getContent = (node) => {
       if (node.type.name === 'text') {
-        if (markdownSchema.marks.link.isInSet(node.marks)) {
-          content.push(node);
+        if (content.length > 0 && node.textContent.length > 0
+          && !node.textContent.match(/^\s+/)
+          && !content.match(/^.+\s$/)) {
+          content += ' ';
```

Comment:
regex rookie confirms: this is just inserting a space in case two nodes don't have one between each other, right?